NMS-14633: Add alt text to Operation (pt. 2) (#5677)
OpenNMS/opennms: docs/modules/operation/pages/deep-dive/alarms/introduction.adoc

= Alarms

{page-component-title} monitors problem states with its managed entities, their resources, the services they provide, and any applications that they host.
Simply stated, {page-component-title} uses the network to monitor problem states.

> While events carry problem state attributes, such as acknowledgement and severity, alarms are the indicators of problems in the network (see xref:deep-dive/bsm/introduction.adoc[]).

Reviewer: ```suggestion
While events carry immutable problem state attributes such as severity, alarms indicate problems in the network (see xref:deep-dive/bsm/introduction.adoc[]) and carry changeable attributes such as acknowledgment.
```
